app/parse.py

```diff
@@ -14,5 +22,39 @@ class Course:
     course_type: CourseType
 
 
+def study_format(course_soup: BeautifulSoup) -> bool:
+    value = course_soup.select_one(
+        ".typography_landingH3__vTjok"
+    ).text.split()[-1]
+
+    return value == "Вечірній"
```

Comment:
Why do you need this separated function if you can look for value in your name attribute below?